### DISCUSSION TITLE
Move "Customise Engine Configuration" into "Tools" and rename

### DISCUSSION THREAD

**Discussion by timothyqiu** [godotengine/godot-proposals#9468] [Editor]
It's a standalone tool that does not affect the project at all.

To avoid confusion, rename it to something like "Engine Compilation Configuration Editor".

**Comment by Calinou** [comment]
I tried implementing this in https://github.com/Calinou/godot/tree/editor-move-build-profile-editor, but the option now does nothing when clicked and I have no idea why (no error is printed).

**Reply by timothyqiu** [reply]
The handling code for `TOOLS_BUILD_PROFILE_MANAGER` should be moved from `_menu_option_confirm()` into `_tool_menu_option()` :P

**Reply by Calinou** [reply]
It works, thanks :slightly_smiling_face: 

I've opened a PR: https://github.com/godotengine/godot/pull/90441
